apps/server/Dockerfile

```diff
@@ -1,22 +1,31 @@
-FROM oven/bun
+# syntax=docker/dockerfile:1.7
+# This file is generated by Dofigen v2.0.0
+# See https://github.com/lenra-io/dofigen
 
-WORKDIR /app
-
-# Copy project
-COPY . .
-
-# To keep the image small ;)
-RUN rm -rf /app/apps/docs
-RUN rm -rf /app/apps/screenshot-service
-RUN rm -rf /app/apps/web
-RUN rm -rf /app/packages/api
-RUN rm -rf /app/packages/integrations/vercel
-#  Temp solution for bun
-RUN bun install --ignore-scripts
+# install
+FROM oven/bun@sha256:eb409bed239c3adff079a6b71283f151e802d66b99f643ba7a71e1be7d3da513 AS install
```

Comment:
@taorepoara  do we really need the hash?

Reply:
The hash gives you the control on the `FROM` images. Dofigen does it by default, but I'll add a flag to avoid that: https://github.com/lenra-io/dofigen/issues/278